README.md

```diff
@@ -0,0 +1,57 @@
+# Docker Poetry
+
+Docker Poetry is a Docker image for Python projects with dependencies managed by Poetry.
+
+![Poetry](https://img.shields.io/badge/Poetry-1.1.13-purple) ![Python](https://img.shields.io/badge/Python-3.10%20(default)%20%7C%203.9%20%7C%203.8%20%7C%203.7-blue) ![OS](https://img.shields.io/badge/OS-Debian%20(default)%20%7C%20Slim%20%7C%20Alpine-orange
+)
+## Why
+
+At <a href="https://withlogic.co/">LOGIC</a> our go-to stack for web applications is; the latest LTS Django on the latest Python using Poetry as our dependency manager in Docker. [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is a core principle for us, so a public Docker image with the right Python and Poetry version is an exceptional base for new projects.
+
+## Usage
+
+Just pick the Poetry version you wish to use and optionally also a Python version and OS.
+
+```
+ghcr.io/parisk/poetry:<poetry_version>-python-<python_version>-<os_variant>
+```
+
+For example, if you want to use Poetry 1.1.13 with Python 3.10 (default) on Debian (default) in your Dockerfile, you would do the following:
+
+```dockerfile
+FROM ghcr.io/withlogicco/poetry:1.1.13
+```
+
+If you do the same, with Python 3.8 on Debian Slim you would do the following:
+
+```dockerfile
+FROM ghcr.io/withlogicco/poetry:1.1.13-python-3.8-slim
+```
+
+## Supported versions
+
+### Poetry
+
+Only the latest version of Poetry is included in each build.
```

Comment:
Shouldn't we include more Poetry versions?

Reply:
We will be building every Poetry version as it is being released, so images will be available for all Poetry releases. They already are actually in the original personal repo: https://github.com/parisk/docker-poetry/pkgs/container/poetry/versions.